resources/qml/delegates/ImageMessage.qml

```diff
@@ -28,21 +28,107 @@ AbstractButton {
     height: width*proportionalHeight
     hoverEnabled: true
 
+    state: img.status != Image.Ready ? "ImageLoading" : (timeline.privacyScreen.active ? "PrivacyScreenVisible" : "PrivacyScreenInvisible")
+    states: [
+        State {
+            name: "ImageLoading"
```

Comment:
Why not combine this with PrivacyScreenVisible?

Reply:
When an image is loading, we currently show a placeholder "no image" icon if the blurhash is not available. I thought that we probably don't want to do that if the privacy screen is active, the image is loaded, but there is no blurhash.

Reply:
Well, it would be nice to cross-fade the blurhash to the image, once the image loads though. But maybe that is too much to ask on this PR :3

Reply:
I think I have fixed this, but I wasn't able to get a blurhash to test with yet

Reply:
Just send an image with Nheko? it generates one :D

Reply:
I tried that, but had no luck. I'm trying with Fluffychat now

Reply:
Then you broke something.

Reply:
Or I did

Reply:
It was me

Reply:
Hah! :p

Reply:
Fixed it